### DISCUSSION TITLE
Thank You :partying_face:

### DISCUSSION THREAD

**Discussion by coopbri** [Coopydood/ultimate-macOS-KVM#26] [General]
Hey @Coopydood, just wanted to say thank you for this repo. I've given tons a shot over the years (starting with https://github.com/foxlet/macOS-Simple-KVM). Most recently used https://github.com/kholia/OSX-KVM, and, while great, started to run into issues with some of the more advanced use cases but community discussion was disabled on it. The docs and website here are amazing and -- I can't believe I'm saying this -- thank you for keeping GH discussions and GH issues enabled :joy: 

Cheers! :clinking_glasses:

**Comment by Coopydood** [comment]
Hi, thank you so much for the compliment, it's made my day! Im glad the docs and open support model of the project is noticed. If you ever need any help, you're more than welcome to ask. 

Thanks again!
